Octokit/Clients/TeamDiscussionsClient.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+
+namespace Octokit
+{
+    /// <summary>
+    /// A client for GitHub's Organization Teams API.
+    /// </summary>
+    /// <remarks>
+    /// See the <a href="http://developer.github.com/v3/orgs/teams/">Organization Teams API documentation</a> for more information.
+    /// </remarks>
+    public class TeamDiscussionsClient : ApiClient, ITeamDiscussionsClient
+    {
+        private static readonly string AcceptHeader = AcceptHeaders.Concat(AcceptHeaders.TeamDiscussionsApiPreview, AcceptHeaders.ReactionsPreview);
+
+        /// <summary>
+        /// Initializes a new GitHub Orgs Team API client.
+        /// </summary>
+        /// <param name="apiConnection">An API connection.</param>
+        public TeamDiscussionsClient(IApiConnection apiConnection)
+            : base(apiConnection)
+        {
+        }
+
+        public Task<TeamDiscussion> Get(int teamId, int number)
```

Comment:
XmlComments are missing from all the methods in this class (just copy the ones from the interface)